.env.development

```diff
@@ -4,3 +4,10 @@ NEXT_PUBLIC_REDIRECT_URI=https://next.egghead.io
 NEXT_PUBLIC_DEPLOYMENT_URL=http://localhost:3000
 NEXT_PUBLIC_ALGOLIA_KEY=d1b3f68acf6c2817900630bc0ac6c389
 NEXT_PUBLIC_ALGOLIA_APP=78FD8NWNJK
+NEXT_PUBLIC_DISCORD_AUTHORIZE=https://discord.com/api/oauth2/authorize?client_id=417792875355701249&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Fdiscord%2Fcallback&response_type=code&scope=identify%20email%20guilds%20guilds.join
```

Comment:
two secrets are required for this to work, the `DISCORD_CLIENT_SECRET` and the `DISCORD_BOT_TOKEN` - the rest of these are fine public.